extras/create_send_notifications.py

```diff
@@ -42,26 +43,38 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Build notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    get a notification PDF for CYHY report types.
+    """
+    notifications_to_generate = set()
+    cyhy_parent_ids = set()
+    ticket_owner_ids = db.notifications.distinct("ticket_owner")
+    for request in db.RequestDoc.collection.find({"_id": {"$in": ticket_owner_ids}, "report_types": "CYHY"}, {"_id":1}):
+        notifications_to_generate.add(request["_id"])
+        cyhy_parent_ids = cyhy_parent_ids | find_cyhy_parents(db, request["_id"])
+        print("cyhy_parent_ids: %s" % cyhy_parent_ids)
+    notifications_to_generate.update(cyhy_parent_ids)
+    return notifications_to_generate
+          
+def find_cyhy_parents(db, org_id):
+    """Find CYHY parents.
+
+    Find weekly report types for CYHY parents
+    recursively using the parent IDs.
```

Comment:
The current comment doesn't really describe what this function does.  I suggest this instead:
```suggestion
    """Return CyHy parents of an organization.

    Recursively find all CyHy parents of
    of an organization using parent IDs.
```